crates/services/consensus_module/poa/src/service_test/trigger_tests.rs

```diff
@@ -194,6 +200,60 @@ async fn interval_trigger_produces_blocks_periodically() -> anyhow::Result<()> {
     Ok(())
 }
 
+// In case of error during block creation, the block should be retried after a delay of 1 second.
+#[tokio::test(start_paused = true)]
+async fn retry_block_creation_in_case_error() -> anyhow::Result<()> {
+    let config = Config {
+        trigger: Trigger::Interval {
+            block_time: Duration::new(2, 0),
```

Comment:
Shouldn't this test be that it triggers again after `2` seconds?

Reply:
With @xgreenx we decided that we will not wait `block_time` to recreate a block but a 1 second hard number